appender-core/src/main/java/com/van/logging/aws/AwsClientHelpers.java

```diff
@@ -63,11 +68,19 @@ static AwsClientBuilder getS3ClientBuilder() {
     public static AmazonS3 buildClient(
         String accessKey, String secretKey, String sessionToken,
         Region region,
-        String serviceEndpoint, String signingRegion
+        String serviceEndpoint, String signingRegion, boolean pathStyleAccess
     ) {
         AWSCredentialsProvider credentialsProvider =
             getCredentialsProvider(accessKey, secretKey, sessionToken);
-        AwsClientBuilder builder = getS3ClientBuilder();
+        AwsClientBuilder builder = null;
```

Comment:
If you are really functional-driven (I'm not), you can also try:
```
AwsClientBuilder builder = getS3ClientBuilder();

builder = builder.withPathStyleAccessEnabled(Boolean.valueOf(pathStyleAccess));
...
```

Reply:
i tried but the signature is not showing on builder instance.
Please find the attached SS. 
![Screenshot from 2020-12-30 12-33-59](https://user-images.githubusercontent.com/62426428/103335575-50263680-4a9b-11eb-8834-503f791068fd.png)

Reply:
Oh. That's because the current class is too generalized. Change `AWSClientBuilder` to `AmazonS3ClientBuilder` in the `AwsClientHelpers.java` file and it'll work. The change is fine since the context is specific to S3, so we're not losing any abstractions.

```
import com.amazonaws.services.s3.AmazonS3ClientBuilder;

...

    static AmazonS3ClientBuilder getS3ClientBuilder() {
        return AmazonS3ClientBuilder.standard();
    }

...
    public static AmazonS3 buildClient(
        String accessKey, String secretKey, String sessionToken,
        Region region,
        String serviceEndpoint, String signingRegion
    ) {
        ...
        AmazonS3ClientBuilder builder = getS3ClientBuilder();
        ...
        builder.setPathStyleAccessEnabled(Boolean.valueOf(pathStyleAccess));
```

There may also be some changes needed in some unit tests, but that should do it.